src/ui/map.js

```diff
@@ -818,6 +818,30 @@ class Map extends Camera {
     }
 
     /**
+     * Add an image to the style. This image can be used in `icon-image`,
+     * `background-pattern`, `fill-patern`, and `line-pattern`.
+     *
+     * @param {string} name The name of the image.
+     * @param {HTMLImageElement|ArrayBufferView} image The image as an `HTMLImageElement` or `ArrayBufferView` (using the format of [`ImageData#data`](https://developer.mozilla.org/en-US/docs/Web/API/ImageData/data))
+     * @param {Object} [options] Only required if passing an `ArrayBufferView`
+     * @param {number} options.width The pixel width of the `ArrayBufferView` image
+     * @param {number} options.height The pixel height of the `ArrayBufferView` image
+     * @param {Object} options.pixelRatio The ratio of pixels in the `ArrayBufferView` image to physical pixels on the screen
```

Comment:
`{Object}` ⇢ `{number}`. Also, should this be marked as optional? It looks like the implementation will default to screen pixelRatio if it's not provided.